pkg/controller/migplan/registry_health.go

```diff
@@ -0,0 +1,143 @@
+package migplan
+
+import (
+	"context"
+	"github.com/konveyor/mig-controller/pkg/apis/migration/v1alpha1"
+	"github.com/konveyor/mig-controller/pkg/compat"
+	health "github.com/konveyor/mig-controller/pkg/health"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/client-go/util/workqueue"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	k8sclient "sigs.k8s.io/controller-runtime/pkg/client"
+	event2 "sigs.k8s.io/controller-runtime/pkg/event"
```

Comment:
can probably remove the alias on this import?

Reply:
Done